Assets/Scripts/Demo/Actions/CharacterCrouch.cs

```diff
@@ -16,15 +16,20 @@ public class CharacterCrouch : CharacterAction
         private Controller2D characterController2D;
         protected ControllerInputModule input;
 
-        private void Start()
+        public override void Awake()
         {
-            input = controllableCharacter.InputModule();
+            base.Awake();
             boxCollider2D = GetComponent<BoxCollider2D>();
             characterController2D = GetComponent<Controller2D>();
             defaultSize = boxCollider2D.size;
```

Comment:
This 2 values are assigned but never used, I think is safe to remove them.

Reply:
I removed this action as is not complete, maybe in a future update I can add a working crouch action.